docs/rules.md

```diff
@@ -559,3 +559,72 @@ parameters:
     checkConfigTypes: true
 ```
 
+## NoModelForwardingToBuilder
+
+This rule checks for calling methods on an `Illuminate\Database\Eloquent\Model` instance that are actually forwarded to a Builder instance. It helps prevent unexpected behaviors like executing `first()`, `get()` on already fetched models.
+
+### Examples
+
+Given the following Eloquent model:
+
+```php
+class Post extends Model {}
+```
+
+The following code would result in refetching all posts from the database which is not intended usage:
```

Comment:
For both of these rules, it would be helpful to show what the error message is (see above for an example), you can include the message from the test (with the tip)